draft-ietf-taps-interface.md

```diff
@@ -138,39 +138,38 @@ and potential transport protocols to select from.
 
 # Introduction
 
-This document specifies a modern abstract programming interface atop the
+This document specifies a modern abstract programming interface (API) atop the
```

Comment:
Should be /abstract Application/ if this to define API, as per MW's comment to abstract.